cyberark/conjur
# UX Updates

Closes:
- [GH Issue 325](https://github.com/cyberark/conjur/issues/325)
- [GH Issue 302](https://github.com/cyberark/conjur/issues/302)
- [GH Isue 322](https://github.com/cyberark/conjur/issues/322)
- [GH Isue 329](https://github.com/cyberark/conjur/issues/329)
- [GH Issue 323](https://github.com/cyberark/conjur/issues/323)
- [GH Issue 337](https://github.com/cyberark/conjur/issues/337)

#### What does this pull request do?
- Updates design to reflect more of CYBR branding
- Fixes Safari bug rendering CTA boxes
- Favicon updated to use CYBR icon
- Fixes typo on /install-conjur (Prerequisites)
- Incorporates feedback from Marketing
- Adds Google Tag Manager script & classes

#### What background context can you provide?
This PR is largely an update to incorporate feedback from Marketing and other stakeholders.

#### Where should the reviewer start?
Homepage

#### How should this be manually tested?
- Check that the branding is updated (refer to screenshot)
- Check in Safari that the CTA boxes (Install Conjur & Try Conjur) are rendered correctly
- Confirm that the "Why Conjur" page is removed (this content is moving to cyberark.com)
- On /install-conjur.html confirm that "Prerequisites" is spelled correctly
- Favicon is updated to use CYBR logo (may need to empty cache to see this)

#### Screenshots (if appropriate)
![conjur_org_updates](https://user-images.githubusercontent.com/123787/29975215-91b67724-8f03-11e7-9b2d-ab7cd547f4c7.png)

#### Link to build in Jenkins (if appropriate)
https://jenkins.conjur.net/job/cyberark--conjur/job/ux-updates-04/
#### Questions:
> Does this have automated Cucumber tests?
no

> Can we make a blog post, video, or animated GIF out of this?
no

> Is this explained in documentation?
no

> Does the knowledge base need an update?
no
